lib/emscripten/src/emscripten_target.rs

```diff
@@ -634,6 +634,11 @@ pub fn invoke_iij(ctx: &mut Ctx, index: i32, a1: i32, a2: i32, a3: i32) -> i32 {
     invoke!(ctx, dyn_call_iij, index, a1, a2, a3)
 }
 
+pub fn invoke_iji(ctx: &mut Ctx, index: i32, a1: i32, a2: i32, a3: i32) -> i32 {
```

Comment:
Any reference to the definition of this syscall? I've been looking for it on internet and found no clue... :-/

Reply:
Is not a syscall. The dyncalls are the way emscripten wraps functions to be able to try/catch them natively (either via js, or Rust in our case).